logistics/scripts/modules/settings/settings-management.psm1

```diff
@@ -324,9 +324,29 @@ function New-DevelopmentAppSettings([hashtable] $Settings = @{ }) {
 
         $credentialSettings = Merge-Hashtables @{ }, (Get-CredentialSettingsByProject)[$project]
         $newUserSettingsPath = Join-Path $projectPath "appsettings.user.json"
```

Comment:
these user files should not be generated anymore

Reply:
I changed to appsettings.secret.json .  Once I added to APPDATA folder using same json file and I am deleting it immediately in next line now .

Reply:
yes , I am not generating user files now .